pkg/remote/aws/sns_topic_subscription_supplier.go

```diff
@@ -71,12 +88,22 @@ func (s *SNSTopicSubscriptionSupplier) Resources() ([]resource.Resource, error)
 }
 
 func (s *SNSTopicSubscriptionSupplier) readTopicSubscription(subscription *sns.Subscription, alertr alerter.AlerterInterface) (cty.Value, error) {
-	if subscription.SubscriptionArn != nil && *subscription.SubscriptionArn == "PendingConfirmation" {
-		alertr.SendAlert(
-			fmt.Sprintf("%s.%s", aws.AwsSnsTopicSubscriptionResourceType, *subscription.SubscriptionArn),
-			&pendingTopicAlert{subscription.Endpoint},
-		)
-		return cty.NilVal, nil
+	if subscription.SubscriptionArn != nil && !arn.IsARN(*subscription.SubscriptionArn) {
+		switch *subscription.SubscriptionArn {
+		case "PendingConfirmation":
```

Comment:
yeah I wondered if we should, the alert is more generic but as I added the wrong arn and it will say "PendingConfirmation" I think it could already give a hint

Reply:
Yeah the generic one is enough. I'm wondering about completely remove the alert, I don't see any value for the end user there

Reply:
You could have subscription not working that you don't see at all or see them unmanaged the alert can probably help you get what's happening ?

Reply:
yeah so let's go for a single generic alert

Reply:
Did that in the last commit :+1: